packages/terra-site/package.json

```diff
@@ -33,7 +33,7 @@
     "terra-clinical-item-view": "^0.x",
     "terra-content": "^0.x",
     "terra-detail-view": "^0.x",
-    "terra-grid": "^1.0.0",
+    "terra-grid": "^2.0.0",
```

Comment:
Good call. Versioning is a bit odd here. `terra-grid v1.0.0` does not contain react components. So while `terra-detail-view` denotes a dev dependency on `terra-grid: ^1.0.0` it was really referencing the code in terra-grid that would become `terra-grid v2.0.0` via npm-link. I'm open to talking about this more in our bi-weekly meetings. There is some functionality around our release script that led us to using this strategy.